step.yml

```diff
@@ -5,9 +5,8 @@ summary: |-
 description: |-
 
   CocoaPods is a dependency manager for Swift and Objective-C projects. This Step uses CocoaPods' `pod install` command to install your dependencies on the virtual machine where your Bitrise build runs.   
-  CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If the version is specified in the Podfile.lock file and the version range specified in the Gemfile.lock file  
-  contains that version, then that version will be installed. Otherwise, the Cocoapods version in the Podfile.lock will be installed as a global gem.  
-  If no Cocoapods version is defined in the Podfile.lock, the preinstalled sytem Cocoapods version will be used.
+  CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If your Gemfile.lock file contains the `cocoapods` gem then the step will call `pod install` with `bundle exec`. 
```

Comment:
```suggestion
  CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file. If your Gemfile.lock file contains the `cocoapods` gem then the step will call `pod install` with `bundle exec`. Otherwise, the Cocoapods version in the Podfile.lock will be installed as a global gem.
```